### PR TITLE
Add OpenCascade.js

### PR DESCRIPTION
**Pull request by donalffons**
Great website, thanks a lot for building an maintaining it!

With this PR, I would like to add `OpenCascade.js` - a port of the OpenCascade CAD library that can be used via WebAssembly.

### CODE REVIEW

**Review by torch2424 — CHANGES_REQUESTED** [review]
> Great website, thanks a lot for building an maintaining it!

You are welcome! Thank you for checking it out, and opening a PR! 😄 🎉 

> With this PR, I would like to add OpenCascade.js - a port of the OpenCascade CAD library that can be used via WebAssembly.

Woahhhh! That's an awesome project! 😄 Thanks for submitting it! 🎉 

---

So I only have one quick change request, it looks like your logo URL is broken, once that is fixed, I'll approve and merge! 😄 🎉

File: showcase/opencascadejs.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+name: OpenCascade.js
+logo_url: https://github.com/donalffons/opencascade.js/blob/master/images/logo%20no%20text.svg
```
[the remark this answers is not in the document]
Comment (donalffons) [reply showcase/opencascadejs.md]: Thanks for the catch! I just pushed a fix.

**Review by donalffons — COMMENTED** [review]
fixed url

**Review by torch2424 — APPROVED** [review]
Awesome! Thank you very much for the submission! 😄 🎉

**Pull request merged**